app/controllers/comments_controller.rb

```diff
@@ -4,8 +4,35 @@ def new
   end
 
   def create
-    @comment = Comment.new(id: params[:id], author_id: current_user.id, Text: params[:text])
-    @comment.save
-    redirect_to '#'
+    @comment = Comment.new(comment_params)
+    @comment.author_id = current_user.id
+    @comment.post_id = params[:post_id]
+    if @comment.save
+      redirect_to user_post_path(user_id: current_user.id, id: params[:post_id])
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    # @post = Post.find_by_id(params[:id])
+    # @user = User.find_by_id(params[:user_id])
+    # @comment = Comment.where(author: @user, post: @post)
```

Comment:
- [ ] since these codes are commented,I would suggest that you remove them to keep your codebase clean and neat as the developer best practices say  :sunglasses: